src/components/RabbitMQServerReleaseInfo/index.js

```diff
@@ -281,12 +281,6 @@ export function RabbitMQServerReleaseInfoTable() {
             "release-eos-community"
           ].join(' ')}>End of Community Support</div>
 
-          <div className={[
-            "release-info-header",
-            "release-eos",
-            "release-eos-commercial"
-          ].join(' ')}>End of Extended Commercial Support</div>
-
           {rows}
```

Comment:
The reason for this change is to remove the potentially confusing information regarding the fixed term extended commercial support in this table. The Broadcom approach is more flexible in that we are taking customer requirements for extended commercial support into account on a case by case basis.